sequencer.c

```diff
@@ -4374,11 +4377,9 @@ int check_todo_list(void)
 	return res;
 }
 
-int append_todo_help(unsigned edit_todo, unsigned keep_empty)
+static void append_todo_help(unsigned edit_todo, unsigned keep_empty,
+			     struct strbuf *buf)
```

Comment:
I think it would be easier to review if changes to this function were in a separate commit.